server/services/interventionsService.test.ts

```diff
@@ -1039,6 +1064,34 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
 
+  describe('assignSentReferral', () => {
```

Comment:
Are there any validation failures that we might need to handle?

Reply:
We don't expect the request ti result in any validation failures

On another note, we would like the id for the assignedTo user so it can be created/update in the database as a result of the request.

Reply:
Do we not want a test for if there wasn't no user found with that username/id?

Reply:
> On another note, we would like the id for the assignedTo user so it can be created/updated in the database as a result of the request.

Added `userId` to the `AuthUser` type.

Reply:
> Do we not want a test for if there's no user found with that username/id?

I don't think we need it - we don't have any other tests of a 404 response in this file.